src/components/molecules/GridTable.tsx

```diff
@@ -1,97 +1,61 @@
-import { Column } from "primereact/column";
-import { FC, useEffect, useState } from "react";
+import { FC } from "react";
+import CustomCard from "../atoms/CustomCard";
 import CustomImage from "../atoms/CustomImage";
-import CustomTable from "../atoms/CustomTable";
-import FilledCheckCircle from "../../assets/filled_check_circle.svg";
-import Warning from "../../assets/warning.svg";
-import { SEVERITY } from "../../helpers/constants/appConstants";
 import { GridTableProps } from "../../@types/components/commonTypes";
+import { getSeverityStyles } from "../../helpers/utils/utils";
 
 const GridTable: FC<GridTableProps> = (props) => {
-  const [rowDataList, setRowDataList] = useState<any[]>([]);
+  const { className = "", dataClassName = "" } = props;
+  const colWidth = `grid-cols-${props.columns}`;
```

Comment:
Provide a default value for columns
const colWidth = `grid-cols-${props.columns || 1}`;